web/src/lib/stores/preferences.store.ts

```diff
@@ -46,3 +46,5 @@ export interface AlbumViewSettings {
 export const albumViewSettings = persisted<AlbumViewSettings>('album-view-settings', {
   sortBy: 'Most recent photo',
 });
+
+export const lastUpdatedProfilePicture = persisted<number>('last-updated-profile-picture', 0, {});
```

Comment:
What is the purpose of this store?

Reply:
I think to bypass the cache for the profile picture, although I'm sure there is a better way.

Reply:
The profile picture is loaded via an static url and by that gets cached by the browser. When ever the profile picture gets changed in the web, the store gets updated and accordingly the static url. Resulting in an updated profile picture.

Reply:
Just add the 'no-cache' cache-control header to the response. The browser will still cache the asset data, but will revalidate the request against the etag. Saves some code and is what we intend to do for all the API endpoints at some point.